app/code/Magento/Developer/Console/Command/TemplateHintsStatusCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+
+/**
+ * Command to show frontend template hints status
+ */
+class TemplateHintsStatusCommand extends Command
+{
+
+    /**
+     * command name
+     */
```

Comment:
Comments are not necessary if it's exactly the same as const name
![image](https://user-images.githubusercontent.com/1639941/72825273-09b73d80-3c77-11ea-8062-dfcd62402d79.png)

Reply:
I've copied from magento2/app/code/Magento/Developer/Console/Command/TemplateHintsEnableCommand.php (Core)